examples/ts/fetch-tickers/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "fetch-tickers",
+  "version": "0.1.0",
+  "description": "ccxt example code in typescript.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "files": [
+    "build/src"
+  ],
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ccxt/ccxt.git"
+  },
+  "keywords": [],
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "check": "gts check",
+    "clean": "gts clean",
+    "compile": "tsc -p .",
+    "fix": "gts fix",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "posttest": "npm run check",
+    "start": "node ./build/index"
+  },
+  "dependencies": {
+    "ansicolor": "^1.1.92",
+    "as-table": "^1.0.55",
+    "ccxt": "^1.18.1145",
+    "ololog": "^1.1.146"
+  },
+  "devDependencies": {
+    "gts": "^1.1.0",
```

Comment:
We are a bit paranoid on adding deps, tbh ) Do you absolutely insist on adding `gts`? Is it commonplace in the TS world? (I'm not too familiar with it).

Reply:
No, not at all. It was the first time I actually used gts. :) (I was just looking for a quick/_simplest_ scaffolding tool for a ts project, and I ended up with that.) I'll remove the dependencies. Thanks.